index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-WK01NEHZ12"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){window.dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        </script>
+        <meta charset="utf-8">
+        <title> Carlos Philippe </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#"><img src="images/logo.png" alt="my logo" ></a>
+  <nav>
+    <class="hamburger"><img src="images/Union.png" alt="Menu">
+  </nav>
+</header>
+        <section id="home_presentation">
+          <h1 class="short-presentation">I'm Carlos
+
+           I Glad to see you! </h1>
+  <p>
+  I’m a web developer and web master! I can help you build a product ,
+  feature or website Look through some of my work and experience!
+  If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+           <footer>
+                 <h4>LET'S CONNECT</h4>
+                 <div id="contact">
+                   <ul>
+                      <li class="icon"><a href="#" target="_blank"><img src="images/twitter.svg" alt="github link"></a></li>
+                     <li class="icon"><a href="#" target="_blank"><img src="images/Linkedin icon-1.svg" alt="linked link"></a></li>
+                     <li class="icon"><a href="#" target="_blank"><img src="images/model.svg" alt="Angel list link"></a></li>
+                     <li class="icon"><a href="#" target="_blank"><img src="images/github.svg" alt="github link"></a></li>
+                     <li class="icon"><a href="#" target="_blank"><img src="images/Angel.svg" alt="Angel list link"></a></li>
```

Comment:
- [ ] Kindly give your social links a `margin-top` and align them to the left to match the Figma design. Also adjust the `let's connect` text to match the original design

![image](https://user-images.githubusercontent.com/98453979/199342521-4ace0a38-4bc4-4acb-9c8c-4b749c8a631e.png)

### Checkout Figma design

![image](https://user-images.githubusercontent.com/98453979/199342707-f65c529d-70c2-4469-9ae1-d7182e4881a1.png)